File: app/src/main/java/jp/co/pannacotta/lunch_app/model/Shop.java
package jp.co.pannacotta.lunch_app.model;

import com.google.gson.annotations.SerializedName;

import org.parceler.Parcel;

@Parcel
public class Shop {
    public String lat;
    public String lng;
    public String lunch;
    public String name;
    public Photo photo;
    public Urls urls;
    @SerializedName("catch")
    public String catchCopy;
    //public Genre genre;
    //public String access;
    //public String free_food;
    // public String private_room;
}

File: app/src/main/java/jp/co/pannacotta/lunch_app/model/Shop.kt
package jp.co.pannacotta.lunch_app.model

import com.google.gson.annotations.SerializedName
import org.parceler.Parcel

@Parcel
class Shop {
    @JvmField
    var lat: String? = null
    @JvmField
    var lng: String? = null
    @JvmField
    var lunch: String? = null
    @JvmField
    var name: String? = null
    @JvmField
    var photo: Photo? = null
    @JvmField
    var urls: Urls? = null

    @JvmField
    @SerializedName("catch")
    var catchCopy: String? = null
    //public Genre genre;
    //public String access;
    //public String free_food;
    // public String private_room;
}